packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.controller.js

```diff
@@ -71,6 +68,8 @@ export default class TelecomPackMigrationCtrl {
           // Retrieve line associated to the service
           return this.getLine(service);
         }
+        // Display offers
+        this.process.currentStep = 'offers';
```

Comment:
Since it's used multiple times here, maybe we can consider using a constant.
see: `packages/manager/apps/telecom/src/app/telecom/pack/migration/pack-migration.constant.js`